drivers/ipm/Kconfig

```diff
@@ -55,6 +55,12 @@ config XLNX_IPI
 	  Inter Processor Interrupt driver for AMD-Xilinx
 	  platforms such as ZynqMP Ultrascale+.
 
+config IPM_SEND_FW_READY_REPLY
+	hex "Send a FW_READY reply message"
+	default 0xFFFF
+	help
+	  Send FW_READY reply to check for FW boot completion
+
```

Comment:
If you want you can move this into `Kconfig.imx`, that would be acceptable.

Reply:
I added this new version as @arnopo suggested -  just for all of us to review it :)
But, I prefer the first version of this PR - adding a config just for IPM_IMX.
Since the fw_ready is waited only for IMX we should fix this limitation only for imx.

@arnopo what do you think?

Reply:
I reverted to my previous version (with small changes on commit message and CONFIG description).

I added the config only for _i.MX family_.
I enable it only for the board I need (right now this is only for the `openamp_rsc_table` sample compiled for `nxp_adsp_imx8m`)